config.lua

```diff
@@ -40,6 +40,31 @@ box.once("init", function()
     })
     st:truncate()
 
```

Comment:
Should we use a separate `config.lua` for benchmarking? We can disable WAL there for better performance https://www.tarantool.io/en/doc/latest/reference/configuration/#cfg-binary-logging-snapshots-wal-mode

Reply:
AFAIU WAL is disabled when we create a temporary space. (Option `temporary` is true). Found it [here.](https://www.tarantool.io/en/doc/latest/reference/reference_lua/box_schema/space_create)